onnxscript/tests/function_libs/torch_aten/ops_correctness_test.py

```diff
@@ -402,25 +430,17 @@ def _where_input_wrangler(
 TESTED_OPS = frozenset(OPINFO_FUNCTION_MAPPING)
 
 EXPECTED_SKIPS_OR_FAILS = (
-    *(
-        # ONNX Runtime 1.13 skips
-        (
-            xfail("logsumexp", reason="ONNX Runtime 1.13 does not support ReduceLogSumExp-18"),
-            xfail(
-                "nn.functional.upsample_nearest2d",
-                reason="ONNX Runtime 1.13 does support opset18",
-            ),
-        )
-        if version_utils.onnxruntime_older_than("1.14")
-        else ()
+    xfail(
+        "logsumexp",
+        reason="ONNX Runtime 1.13 does not support ReduceLogSumExp-18",
+        active_if=version_utils.onnxruntime_older_than("1.14"),
```

Comment:
Do we need this or should we just remove the xfail?

Reply:
For now we still need it because the ort version with function default attribute support is still a variant of 1.13. @gramalingam i think we can move to use nightly after liqun s PR?